app/editor/src/components/upload/Upload.tsx

```diff
@@ -129,4 +130,6 @@ export const Upload: React.FC<IUploadProps> = ({
       />
     </styled.Upload>
   );
+};
```

Comment:
Formatting looks invalid, this will break linting

Reply:
Fixed it.